docs/openapi/schemas.yaml

```diff
@@ -438,3 +437,167 @@ AuditList:
+    partition: 0
+    offset: 1234
+    value:
+      content: |-
+        ewogICAgImV4dGVybmFsX3JlcXVlc3RfaWQiOiAiMTIzNDUiLAogICAgInJlcXVlc3Rvcl9pZCI6ICIxMjM0NTY3ODkw
+        QUJDREVGMTIzNDU2NzhAYWRvYmUuY29tIiwKICAgICJvd25lcl9pZCI6ICIxMjM0NTY3ODkwQUJDREVGMTIzNDU2NzhA
+        QWRvYmVPcmciLAogICAgInJlcXVlc3RfdHlwZSI6ICJSRUdVTEFSIiwKICAgICJyZXF1ZXN0b3Jfc3lzdGVtIjogIkFB
+        VUkiLAogICAgImZ1bGZpbGxtZW50X2lkIjogIjEyMzQ1Njc4OTBhYmNkZWYxMjM0NTY3ODkwYWJjZGVmMTIzNDU2Nzg5
+        MGFiY2RlZjEyMzQ1Njc4OTBhYmNkZWYiLAogICAgImNyZWF0ZV9kYXRlIjogIjIwMjQtMDEtMjJUMDk6MjA6MDEuMTIz
+        NDVaIiwKICAgICJpdGVtcyI6IFsKICAgICAgICB7CiAgICAgICAgICAgICJleHRlcm5hbF9pdGVtX2lkIjogIjEyMzQ1
+        LWFiY2QtMTIzNC0xMjM0LTEyMzQ0MzIxIiwKICAgICAgICAgICAgInF1YW50aXR5IjogIlVOTElNSVRFRCIsCiAgICAg
+        ICAgICAgICJvZmZlcl9pZCI6ICIxMjM0NTY3ODkwQUJDREVGMTIzNDU2Nzg5MEFCQ0RFRiIsCiAgICAgICAgICAgICJh
+        Y2NlcHRlZF90ZXJtcyI6IHsKICAgICAgICAgICAgICAgICJjb250cmFjdF9pZCI6ICIxMjM0NTY3ODkwQUJDREVGMTIz
+        NCIsCiAgICAgICAgICAgICAgICAiYWNjZXB0ZWRfYWdyZWVtZW50IjogIklOSVRJQUwiCiAgICAgICAgICAgIH0sCiAg
+        ICAgICAgICAgICJmdWxmaWxsYWJsZV9pdGVtc19jb21wbGV0ZWQiOiBbCiAgICAgICAgICAgICAgICB7CiAgICAgICAg
+        ICAgICAgICAgICAgImlkIjogIjEyM2U0NTY3LWU4OWItMTJkMy1hNDU2LTQyNjYxNDE3NDAwMCIsCiAgICAgICAgICAg
+        ICAgICAgICAgImNvZGUiOiAiZHhfZXhhbXBsZV9zb2x1dGlvbiIsCiAgICAgICAgICAgICAgICAgICAgImZ1bGZpbGxt
+        ZW50X2RldGFpbHMiOiB7CiAgICAgICAgICAgICAgICAgICAgICAgICJmdWxmaWxsYWJsZV9lbnRpdHlfcmVzb3VyY2Vf
+        bG9jYXRvciI6ICJodHRwczovL2V4YW1wbGUuY29tLzEyMzQ1Njc4OTBBQkNERUYxMjM0NTY3OEBBZG9iZU9yZyIsCiAg
+        ICAgICAgICAgICAgICAgICAgICAgICJmdWxmaWxsYWJsZV9lbnRpdHlfcmVzb3VyY2VfbmFtZSI6ICJFeGFtcGxlIFNv
+        bHV0aW9uIgogICAgICAgICAgICAgICAgICAgIH0KICAgICAgICAgICAgICAgIH0KICAgICAgICAgICAgXQogICAgICAg
+        IH0KICAgIF0KfQ==
+      headers:
+        x-request-id:
+          - "123e4567-e89b-12d3-a456-426614174000"
+        miso-trace-id:
+          - "12345678abcdef12:12345678abcdef12:12345678abcdef12:1"
+        com-adobe-hoolihan-source-partition:
+          - "0"
+        com-adobe-hoolihan-source-offset:
+          - "1234"
+        com-adobe-hoolihan-publish-time:
+          - "1705915000000"
+        x-api-key:
+          - "example_publisher_client_id"
+        com-adobe-hoolihan-publisher:
+          - "example_publisher_client_id"
+        com-adobe-hoolihan-source-topic:
+          - "example_source_topic"
+        com-adobe-hoolihan-on-behalf-of:
+          - "example_other_client_id"
+        Content-Type:
+          - "application/json"
+HoolihanEvents:
+  type: array
+  items:
+    $ref: '#/HoolihanEvent'
+ProcessingStatus:
+  type: string
+  enum:
+    - 'accepted'
+    - 'rejected'
+  example: 'accepted'
+FulfillmentProcessingResults:
+  type: array
+  items:
+    type: object
+    required:
+      - status
+    properties:
+      status:
+        description: The processing status (whether or not the event was accepted for processing)
```

Comment:
I would not call it "processing status" but rather something like "acceptance status" (since we're really not returning any status of the subsequent processing)